CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contribution
+
+# How to contribute?
+
+## Steps:
+  
+
+  -  Go to [issues](https://github.com/s-ayush2903/Recipebook/issues) & select one upon which you want to work!
+  
+  -  Fork the [repository](https://github.com/s-ayush2903/Recipebook.git)
+  
+  -  Clone the fork [repo](https://github.com/<your_username>/Recipebook)
+     - git clone https://https://github.com/<your_username>/Recipebook
```

Comment:
Wherever we're using commands, use 'em between a pair of backticks(*``*), that'll make 'em look like code, e.g., `this is a sample how command should appear in md files`

Reply:
Thanks for the feedback. I will change the file accordingly.